decidim-core/lib/decidim/core/engine.rb

```diff
@@ -3,6 +3,8 @@
 require "rails"
 require "active_support/all"
 require "action_view/railtie"
+require "active_model"
+require "active_record"
```

Comment:
Why do we need to add `active_record` here? Shouldn't it be loaded by the railtie?
https://github.com/decidim/decidim/blob/fc6e578c7a848a7ee3fa3ad8e5e15294120bc31e/decidim-core/lib/decidim/rails.rb#L13

https://github.com/rails/rails/blob/d38ff07a39c98f5db2b15ca3a51acee48c972205/activerecord/lib/active_record/railtie.rb#L3

Reply:
For some reason, if we do not explicitly require active model, the test suite is failing with uninitialized constant ActiveModel::Validations::EachValidator

Maybe is something else in the test suite that causes that, but it may be a subject to another PR, addressing explicitly only that topic.

Reply:
Could you try changing this:
https://github.com/decidim/decidim/blob/fc6e578c7a848a7ee3fa3ad8e5e15294120bc31e/decidim-dev/lib/decidim/dev/railtie.rb#L3

To:
```ruby
require "decidim/rails"
```